CHANGELOG.md

```diff
@@ -22,6 +22,14 @@
 - Remove fine-grained features from `fidget` crate, because we aren't actually
   testing the power-set of feature combinations in CI (and some were breaking!).
   The only remaining features are `rhai`, `jit` and `eval-tests`.
+- Add new `ShapeVars<T>` type, representing a map from `VarIndex -> T`.  This
+  type is used for high-level rendering and meshing of `Shape` objects that
+  include supplementary variables
+- Add `Octree::build_with_vars` and `Image/VoxelRenderConfig::run_with_vars`
+  functions for shapes with supplementary variables
+- Change `ShapeBulkEval::eval_v` to take **single** variables (i.e. `x`, `y`,
+  `z` vary but each variables are contant).  Add `ShapeBulkEval::eval_vs` if
+  `x`, `y`, `z` and variables are _all_ changing through the slices.
```

Comment:
This part is just confusing to me, but maybe because it is late at night. But maybe expand a bit somewhere? (The comments on `eval_vs` didn't help me either.)

Reply:
Added some more comments to the `eval_v` and `eval_vs` docstrings